docs/sources/configure/_index.md

```diff
@@ -886,6 +886,28 @@ volume_results_cache:
   # CLI flag: -frontend.volume-results-cache.compression
   [compression: <string> | default = ""]
 
+# Cache instant metric query results.
+# CLI flag: -querier.cache-instant-metric-results
+[cache_instant_metric_results: <boolean> | default = false]
+
+# If a cache config is not specified and cache_instant_metric_results is true,
+# the config for the results cache is used.
+instant_metric_results_cache:
+  # The cache block configures the cache backend.
+  # The CLI flags prefix for this block configuration is:
+  # frontend.instant-metric-results-cache
+  [cache: <cache_config>]
+
+  # Use compression in cache. The default is an empty value '', which disables
+  # compression. Supported values are: 'snappy' and ''.
+  # CLI flag: -frontend.instant-metric-results-cache.compression
+  [compression: <string> | default = ""]
```

Comment:
Why are we disabling compression by default?

Reply:
Actually, this is the auto-generated with `make doc`. Looks like this [default](https://github.com/grafana/loki/blob/kavirajk%2Fcache-instant-queries2/pkg/storage/chunk/cache/resultscache/config.go#L21) for all [other caches ](https://github.com/grafana/loki/blob/e373341ee7f294bdba535f9905543c662ed91de0/docs/sources/configure/_index.md#query_range)as well.